new-stack-templates.json

```diff
@@ -408,6 +408,41 @@
     "title": "WebLogic 14R1 (autosetup): Camunda Platform 7.20.x",
     "type": 2
   },
+  {
+    "categories": [
+      "server"
+    ],
+    "description": "NOTE: Please first start a database container and fill out the database fields before creating this container!",
+    "env": [
+      {
+        "label": "Database Host",
+        "name": "DATABASE_HOST"
+      },
+      {
+        "label": "Database Port",
+        "name": "DATABASE_PORT"
+      },
+      {
+        "default": "XE",
+        "label": "Database Name",
+        "name": "DATABASE_NAME"
+      },
+      {
+        "default": "oracle",
+        "label": "Database Type (oracle, postgres, h2)",
+        "name": "DATABASE_TYPE"
+      }
+    ],
+    "logo": "https://portainer-io-assets.sfo2.digitaloceanspaces.com/logos/registry.png",
+    "note": "List: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-729-latest/docker-stack.yml\">https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-721-latest/docker-stack.yml</a>",
```

Comment:
❌ Looks like a `7.19` copy-paste error that also needs to be corrected for the `7.20` entry above.
```suggestion
    "note": "List: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-721-latest/docker-stack.yml\">https://github.com/camunda/portainer-templates/blob/master/stacks/weblogic14r1-721-latest/docker-stack.yml</a>",
```